HelpSource/Classes/SoundFile.schelp

```diff
@@ -26,6 +26,24 @@ ClassMethods::
 method::new
 Creates a new SoundFile instance.
 
+method:: write
+Writes an array (or nested array) to a path as a sound file.
+
+argument:: pathName
+The path as a link::Classes/String::.
+
+argument:: array
+The data to write. Can be an array of number, or if multichannel, an array of arrays of multichannel.
```

Comment:
It is a little unclear what is expected. Maybe:
"Can be an array of numbers for a mono audio file, or an array of arrays of numbers for a multichannel audio file.

Reply:
God that's awful English!
Fixed.